mllib/src/main/scala/org/apache/spark/ml/optim/aggregator/HuberAggregator.scala

```diff
@@ -81,6 +81,9 @@ private[ml] class HuberAggregator(
   } else {
     0.0
   }
+  // make transient so we do not serialize between aggregation stages
+  @transient private lazy val featuresStd = bcFeaturesStd.value
```

Comment:
I don't think this change makes sense. It is just getting the broadcast, not a big overhead...

Reply:
Yes, this one isn't necessary. `coefficients` looks good. But it doesn't need to be lazy.

Reply:
Thanks for the feedback. I've removed the unnecessary change to featuresStd.

@srowen I tried removing the lazy modifier, but that causes both the unit tests and my test case to fail with the following NPE. I don't understand why.

`org.apache.spark.SparkException: Job aborted due to stage failure: Task 2 in stage 3.0 failed 1 times, most recent failure: Lost task 2.0 in stage 3.0 (TID 11, localhost, executor driver): java.lang.NullPointerException
        at org.apache.spark.ml.optim.aggregator.HuberAggregator.$anonfun$add$3(HuberAggregator.scala:109)
        at org.apache.spark.ml.linalg.SparseVector.foreachActive(Vectors.scala:613)
        at org.apache.spark.ml.optim.aggregator.HuberAggregator.add(HuberAggregator.scala:107)`

Reply:
Oh I get it. You wouldn't want to eagerly evaluate the broadcast as it might eval on the driver. OK I think this is a reasonable solution.